src/integrationTest/java/uk/gov/hmcts/reform/hmc/config/MessageProcessorIT.java

```diff
@@ -239,10 +240,11 @@ void shouldThrowErrorForExceptionFlow() throws JsonProcessingException {
         List<ILoggingEvent> logsList = listAppender.list;
         assertEquals(3, logsList.size());
         assertEquals(Level.INFO, logsList.get(0).getLevel());
-        assertEquals("Message of type ERROR received", logsList.get(0).getMessage());
+        assertEquals("Message of type ERROR received", logsList.get(0).getFormattedMessage());
```

Comment:
MessageType.ERROR  can be used to be in sync with other messages.

Reply:
Good spot. Implemented.